cxx-qt-cli/src/main.rs

```diff
@@ -0,0 +1,35 @@
+// SPDX-FileCopyrightText: 2021 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Be Wilson <be.wilson@kdab.com>
+// SPDX-License-Identifier: MIT OR Apache-2.0
+#![cfg_attr(feature = "absolute-paths", feature(absolute_path))]
+
+use clap::Parser;
+use std::path::PathBuf;
+use cxx_qt_build::GeneratedCpp;
+
+#[derive(Parser)]
+#[clap(author, version, about, long_about = None)]
+struct Cli {
+    /// path to Rust file containing module with #[cxx_qt::bridge] attribute macro
+    #[clap(short, long, value_parser)]
+    input: PathBuf,
+
+    /// directory to output generated C++ files
+    #[clap(short, long, value_parser)]
+    output: PathBuf,
+}
+
+
+fn main() { let cli = Cli::parse();
```

Comment:
formatting looks wrong here, which hints that you need to add the following to the root `CMakeLists.txt`

```cmake
add_test_cargo(cxx_qt_cli "${CMAKE_CURRENT_SOURCE_DIR}/cxx-qt-cli/Cargo.toml" DOCTESTS_ON)
```